src/main/java/com/parasoft/demoapp/service/GlobalPreferencesService.java

```diff
@@ -125,12 +125,12 @@ public GlobalPreferencesEntity updateLabelOverridedStatus(Boolean labelsOverride
         GlobalPreferencesEntity globalPreferencesEntity = null;
         try {
             globalPreferencesEntity = getCurrentGlobalPreferences();
+            globalPreferencesEntity.setLabelsOverrided(labelsOverrided);
         } catch (GlobalPreferencesNotFoundException | GlobalPreferencesMoreThanOneException e) {
             // Generally, can not reach here.
             e.printStackTrace();
         }
 
-        globalPreferencesEntity.setLabelsOverrided(labelsOverrided);
         return updateGlobalPreferences(globalPreferencesEntity);
```

Comment:
Is it reasonable to give updateGlobalPreferences a null variable? If not, maybe move the return line also inside the try block and return something else reasonable.